packages/go-gen/src/schema.rs

```diff
@@ -259,12 +259,14 @@ pub fn documentation(schema: &SchemaObject) -> Option<String> {
 /// If the given type is not a special type, returns `None`.
 pub fn custom_type_of(ty: &str) -> Option<&str> {
     match ty {
+        "Uint64" => Some("string"),
         "Uint128" => Some("string"),
         "Binary" => Some("[]byte"),
         "HexBinary" => Some("Checksum"),
         "Addr" => Some("string"),
         "Decimal" => Some("string"),
         "Decimal256" => Some("string"),
+        "Timestamp" => Some("uint64"),
```

Comment:
Uff, seems like here we a single case in which the Go type and the JSON type are different. See also `type BlockInfo struct {` in wasmvm. This is a bit annoying, especially because between Go and Rust using the full range for u64 as JSON numbers is safe to do.

What about returning a pair here for (Go type, Go JSON annotation). E.g. ("uint64", Some("string")) to get

```
Time    uint64 `json:"time,string"`
```

The other solution is to get rid of the implicit conversion in Go and turn the Time/Timestampt fields into strings. But that's a bit inconvenient for Go development.

Reply:
Added. One thing that is still a bit problematic is something like `a: Vec<Timestamp>`.
This is now generated as `` A []uint64 `json:"a"` ``. I am not exactly sure how this *should* look in Go. I guess it would have to have some custom `MarshalJSON` implementation?

Reply:
I think we should work towards not having the problem. It should just be:
- u64 -> uint64 / JSON number
- Uint64/Timestamp -> string / JSON string

The other option would be to have add Uint64 type in Go that always serializes to strings and stores the data as uint64 internally wich convenient uint64 conversion for Go users. This moves the current logic from the annotation to a type.

Reply:
To me a `Uint64` Go type seems like the best solution. It would be relatively nice to use on the Go side while completely circumventing the problem we have here.

Reply:
Okay, let's do that for Uint64 and Int64! Then we have:

```rust
        "Uint64" => Some("Uint64"),
        "Uint128" => Some("string"),
        "Int64" => Some("Int64"),
        "Int128" => Some("string"),
```

and can remove the annotations code

Reply:
I reverted the annotations code and added these mappings, as well as
```rust
"Timestamp" => Some("Uint64"),
```